app/assets/stylesheets/landing/all.css

```diff
@@ -3,5 +3,6 @@
 *= require foundation
 *= require style
 *= require buttons
-*=require_tree .
+*= require ../foundation_and_overrides
+*= require_tree .
 */
```

Comment:
since you import foundation_and_overrides in homepage now, you probably want to remove `require foundation` from line 3.